modules/platforms/dotnet/Apache.Ignite.Core/Impl/Binary/BinaryReflectiveSerializerInternal.cs

```diff
@@ -142,22 +142,18 @@ bool IBinarySerializerInternal.SupportsHandles
 
             var fields = ReflectionUtils.GetAllFields(type).Where(x => !x.IsNotSerialized).ToList();
 
-            IDictionary<int, string> idMap = new Dictionary<int, string>();
+            var fieldsSet = new HashSet<string>();
 
             foreach (FieldInfo field in fields)
             {
                 string fieldName = BinaryUtils.CleanFieldName(field.Name);
 
-                int fieldId = BinaryUtils.FieldId(typeId, fieldName, converter, idMapper);
```

Comment:
I removed this one, since it was used only for exception message. Correct me if it's still required.